src/beale.jl

```diff
@@ -22,7 +22,7 @@ function beale(args...)
     @NLobjective(
       nlp,
       Min,
-      (-1.5 + x[1] * (1.0 - x[2]))^2 + (-2.25 + x[1] * (1.0 - x[2]^2))^2 + (-2.625 + x[1] * (1.0 - x[2]^3))^2
+      (1.5 + x[1] * (1.0 - x[2]))^2 + (2.25 + x[1] * (1.0 - x[2]^2))^2 + (2.625 + x[1] * (1.0 - x[2]^3))^2
```

Comment:
It was right before, wasn't it? It would be easier if you wrote it exactly as in the paper.